clowdapp.yaml

```diff
@@ -343,7 +349,7 @@ parameters:
   value: 50m
 - name: REDIS_SSL
   description: 'Whether to use secured connection to Redis. Use string values of true or false'
-  value: "true"
+  value: "false"
```

Comment:
This should be changed by the environment. If possible, please keep the secured variant as default (for hardening).

Reply:
Reverted this change as requested